### DISCUSSION TITLE
Steps to upgrade nightly?

### DISCUSSION THREAD

**Discussion by soylentgeek** [DSheirer/sdrtrunk#1946] [Q&A]
Hi everyone,

I noticed there is a new nightly compared to what I am running currently.

How do I upgrade my install? 

Is it as simple as unzipping the new files into the existing directory?

(Windows 11 BTW)

D

**Comment by ghost** [comment]
Nope. All you need to do is clone your playlist. That's it. You should **make backup copies of your final playlist** and backup/email it to yourself as a habit everytime you make big playlist changes. Then you just download the new nightly, unzip it, and run it. Don't even need to touch the old sdrtrunk directory. The playlist file holds everything you need. Your whole PC could be destroyed but if you have a copy of the playlist file you can quickly rebuild your scanner system in no time. Oh and just don't forget to set your java ram in sdrtrunk file if that is something you did already. Since the new sdrtrunk file won't have those java changes.
